webapp/public/locales/fr/main.json

```diff
@@ -508,8 +508,9 @@
   "study.modelization.clusters.matrix.common": "Common",
   "study.modelization.clusters.matrix.tsGen": "TS generator",
   "study.modelization.clusters.matrix.timeSeries": "Séries temporelles",
-  "study.modelization.clusters.matrix.fuelCost": "Fuel Cost",
-  "study.modelization.clusters.matrix.co2Cost": "CO2 Cost",
+  "study.modelization.clusters.matrix.availability": "Disponibilité",
+  "study.modelization.clusters.matrix.fuelCosts": "Fuel Costs",
```

Comment:
Translations to french are missing

Reply:
You translated `costGeneration` with "Coût de génération" so you can translate `fuelCosts` with "Coûts du carburant" and `co2Costs` with "Coûts du CO2" ?